src/constants/roles.ts

```diff
@@ -1,8 +1,13 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
-export default {
+import {ChannelModerationRoles} from 'types/roles';
+import {Dictionary} from 'types/utilities';
+
+const Roles: Dictionary<ChannelModerationRoles> = {
```

Comment:
Ditto here. Dictionary vs `{[key: string]: X}` vs Record is definitely confusing.